recipes/cmaps/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cmaps" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "644ad7b5d45923effe275be02622ba0290d8a2f06de7fc1e2e0c90df66f20c13"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - matplotlib
```

Comment:
I don't think you need `matplotlib` and `numpy` here, so can you please try removing them?

Reply:
Done.